code/modules/flufftext/Hallucination.dm

```diff
@@ -701,9 +770,17 @@ var/list/non_fakeattack_weapons = list(/obj/item/weapon/gun/ballistic, /obj/item
 						halimage = null
 		if("death")
 			//Fake death
-			hal_screwyhud = 1
+			hal_screwyhud = 2
 			SetSleeping(20)
-			spawn(rand(50,100))
+			var/area/area = get_area(src)
+			src << "<span class='deadsay'><b>[mind.name]</b> has died at <b>[area.name]</b>.</span>"
+			if(prob(50))
+				var/list/dead_people
+				for(var/mob/dead/observer/G in player_list)
+					dead_people += G
+				var/mob/dead/observer/fakemob = pick(dead_people)
+				src << "<span class='deadsay'><b>DEAD:[fakemob.name]</b> says, \"rip\"</span>"
```

Comment:
"ha, you died", "welcome X", "I see the toxin got you too", "is the AI malf", "I fucking hate double agents"